### PR TITLE
Address Paul's DISCUSS and COMMENTS

### LINKED ISSUE #14: Paul Wouters comments with DISCUSS
1. The layout is completely broken / wrapped, making the document fairly
unreadable. Can this be fixed somehow ?

2. I don't see a method for syslog using TCP without TLS ? In fact, I am using
that on my personal home router to my collection server, without TLS. Could the
document not simply use:

```text
+--rw (transport)
              |  +--:(udp)
              |  |  +--rw udp
              |  |     +--rw address?   inet:host
              |  |     +--rw port?      inet:port-number
              |  |  +--rw tcp
              |  |     +--rw address?   inet:host
              |  |     +--rw port?      inet:port-number
              |  +--:(tls)
```

3. Also, how do ports and addresses combine. Can I specify  "1.2.3.4:80 and
5.6.7.8:443" ? It looks like that is not the case here?

4. I don't see an option for configuring ratelimits, which are commonly available
as options for syslog services?

5. Could the TLS configuration parameters not use another model (one the IESG
reviewed a little while ago?). It seems odd to define these within the syslog
module. It would be nice if any kind of certificate options with TLS could be
included from another module so if there are no TLS options, this module/RFC
does not require updating?

6. Maybe the same applies for the "signers" section but perhaps that is
differently formatted data signed and unrelated to the TLS layer ?

Other Comments:

7. Related question: Is it one certificate+key that used for the TLS connection as
well as to sign data within the payload of packets?

8. facility-filter seems badly named as it also filters for severity ? Maybe
syslog-filter ?

### PR DESCRIPTION
**Pull request by mjethanandani**
Address issue #14

### CODE REVIEW

File: src/yang/example-syslog-configuration-7.1.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@
 <syslog xmlns="urn:ietf:params:xml:ns:yang:ietf-syslog">
   <actions>
     <console>
-      <facility-filter>
+      <syslog-filter>
```
Comment (xorrkaz) [inline src/yang/example-syslog-configuration-7.1.xml]: I think we should just go with `filter`.  `syslog` is implied.

File: src/yang/ietf-syslog.yang [hunk 2]
```diff
@@ -700,69 +700,106 @@ module ietf-syslog {
...
-                                    type inet:port-number;
-                                    default "6514";
-                                    description
-                                      "TCP port 6514 has been
-                                       allocated as the default port
-                                       for syslog over TLS.";
-                                }
-                                uses tlsc:tls-client-grouping;
+                      }
+                      case tcp {
```
Comment (xorrkaz) [inline src/yang/ietf-syslog.yang]: Nice!  I think we might want some text on this in the draft itself.  We might need to update security considerations, too.

File: src/yang/ietf-syslog.yang [hunk 3]
```diff
@@ -700,69 +700,106 @@ module ietf-syslog {
...
+                          description
+                            "This container describes the TCP transport
+                             options.";
+                          reference
+                            "RFC 5426: Transmission of Syslog
+                                       Messages over UDP";
+                          list tcp {
+                            key "address";
+                            description
+                              "List of TCP based sessions.";
```
Comment (xorrkaz) [inline src/yang/ietf-syslog.yang]: Do we want to emphasize "clear text" here?

**Comment by mjethanandani** [comment]
Since 6587 (cleartext TCP for Syslog) is marked historic, dropping it from the model.

**Review by xorrkaz — APPROVED** [review]
LGTM

**Pull request merged**